bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html

```diff
@@ -16,7 +16,7 @@
 <link rel="stylesheet" type="text/css" href="main/libs/ui.grid/ui-grid.min.css">
 <link rel="stylesheet" type="text/css" href="main/libs/ui.select/select.min.css">
 <link rel="stylesheet" type="text/css" href="main/libs/angular-ui-notification/angular-ui-notification.min.css">
-
+<link rel="stylesheet" type="text/css" href="../style.css">
```

Comment:
This link seems to be incorrect. You don't need the asterixes in index.html, and you don't need to explicitly link to the css sheet. It is already done by default.